ci/run.sh

```diff
@@ -2,18 +2,18 @@
 
 set -ex
 
-echo "toolchain versions\n------------------"
+printf 'toolchain versions\n------------------\n'
```

Comment:
Why printf instead of echo?

Reply:
ShellCheck warns:
```
echo "toolchain versions\n------------------"
     ^-- SC2028: echo may not expand escape sequences. Use printf.
```

Reply:
```suggestion
echo -e "toolchain versions\n------------------"
```
WIll this work?

Reply:
It works because we are using bash, but
```bash
echo -e "toolchain versions\n------------------"
     ^-- SC2039: In POSIX sh, echo flags are undefined.
```

Reply:
I agree that printf is probably more correct.  Especially when we start dealing with BSDs which might use ksh or POSIX sh, or perhaps dash on Debian.

Alternatively we change the instructions to "curl | bash" rather than "curl | sh"